package.json

```diff
@@ -20,13 +20,7 @@
   },
   "devDependencies": {
     "@commitlint/config-conventional": "13.2.0",
-    "@edx/frontend-platform": "1.9.6",
-    "@edx/paragon": "14.5.0",
     "husky": "6.0.0",
-    "lerna": "4.0.0",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
-    "react-router-dom": "5.2.0",
-    "react-test-renderer": "16.14.0"
```

Comment:
These dependencies were previously installed at the root `node_modules`, as when importing these dependencies in `catalog-search`, `utils`, `logistration`, etc. would resolve correctly to the parent `node_modules`, instead of local to that package.

However, this causes many peer dependency issues since they are not explicitly defined in the package.json files of packages like `catalog-search`, etc.

So, instead, we will explicitly move these dependencies to each individual package, as needed, at the expense of no longer having a single source of truth version of `react`, `react-dom`, etc. throughout the monorepo.